common/libs/contract.js

```diff
@@ -46,17 +43,24 @@ export default class Contract {
     return method;
   }
 
-  call(name: string, args: any[]): string {
-    const method = this.getMethodAbi(name);
+  getMethodTypes(method: ABIMethod) {
+    return method.inputs.map(i => i.type);
+  }
+
+  getMethodSelector(method: ABIMethod) {
```

Comment:
Replaced with 3285f529834a41cfbe20e5861cb7499eed6966e6, but still leaving `getMethodSelector` so that it can take in an ABIMethod object.